pkg/services/ngalert/eval/eval.go

```diff
@@ -207,3 +207,22 @@ func (evalResults Results) AsDataFrame() data.Frame {
 	f := data.NewFrame("", fields...)
 	return *f
 }
+
+// ConditionEval executes conditions and evaluates the result.
+func ConditionEval(condition *Condition, now time.Time) (Results, error) {
+	alertCtx, cancelFn := context.WithTimeout(context.Background(), setting.AlertingEvaluationTimeout)
```

Comment:
Should use non-global configuration. Something to keep in mind moving forward, but fine for now I guess

Reply:
good point; I'm not even sure if we want to use the same configuration with the existing alerting. We use some other [constants](https://github.com/grafana/grafana/blob/f45449fae6b68e3ab5351ccf79c3c1ef8c5b9890/pkg/services/ngalert/ngalert.go#L20) that probably we would also like to customise in the future.